raiden/tests/unit/transfer/mediated_transfer/test_mediatorstate.py

```diff
@@ -144,39 +144,41 @@ def test_next_route_amount():
     amount = UNIT_TRANSFER_AMOUNT
 
     channels = factories.make_channel_set(
-        [
+        properties=[
             NettingChannelStateProperties(
                 our_state=NettingChannelEndStateProperties(balance=amount)
             ),
             NettingChannelStateProperties(our_state=NettingChannelEndStateProperties(balance=0)),
             NettingChannelStateProperties(
                 our_state=NettingChannelEndStateProperties(balance=amount)
             ),
-        ]
+        ],
+        token_address=factories.UNIT_TRANSFER_DESCRIPTION.initiator
     )
 
     # the first available route should be used
     chosen_channel = mediator.next_channel_from_routes(
-        channels.get_routes(0), channels.channel_map, amount, timeout_blocks
+        channels.get_routes(), channels.sub_channel_map, amount, timeout_blocks
```

Comment:
Can you explain why now it does not receive parameters for channel identifiers?

Reply:
I was testing get_routes, now i do get_routes_by_index(0), i forgot to bring back the 0